test_that_wikidata_works_with_multiple_brands_one_invalid failing

```
======================================================================
FAIL: test_that_wikidata_works_with_multiple_brands_one_invalid (test_wikimedia_link_issue_reporter.Tests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mateusz/Documents/install_moje/OSM_software/wikibrain_py_package_published/test_wikimedia_link_issue_reporter.py", line 490, in test_that_wikidata_works_with_multiple_brands_one_invalid
    self.assertIsNone(should_be_fine)
AssertionError: <wikibrain.wikimedia_link_issue_reporter.ErrorReport object at 0x7f24aa898d90> is not None
```

Given that https://www.wikidata.org/wiki/Q7501155 is actually defunct, it should not be None, right? And if one would be invalid then error should be still emitted?

@KasperFranz 

from #5

--- a/test_wikimedia_link_issue_reporter.py
+++ b/test_wikimedia_link_issue_reporter.py
@@ -4,7 +4,6 @@ import wikibrain.wikidata_knowledge
 import wikibrain.wikimedia_link_issue_reporter
 import wikibrain
 import wikimedia_connection.wikimedia_connection as wikimedia_connection
-import wikimedia_connection.wikidata_processing as wikidata_processing
 import osm_handling_config.global_config as osm_handling_config
 
 
@@ -482,12 +481,16 @@ class Tests(unittest.TestCase):
         self.assertEqual(None, should_be_fine)
 
     def test_that_wikidata_works_with_multiple_brands_one_invalid(self):
-        # https://www.wikidata.org/wiki/Q53268
+        # https://www.wikidata.org/wiki/Q7501155
         matching_tags = {"brand:wikidata": "Q7501155;Q6746"}
         location = None
         object_description = "fake test object"
-        should_be_fine = self.detector().get_the_most_important_problem_generic(matching_tags, location, "node", object_description)
-        self.assertIsNone(should_be_fine)
+        should_not_be_fine = self.detector().get_the_most_important_problem_generic(matching_tags, location, "node", object_description)
+        self.assertEqual(
+            should_not_be_fine.error_message,
+            'wikidata Q7501155 is marked as dissolved',
+            'Shopko is reported as being active but it should be reported as being dissolved'
+        )
 
     def test_get_dissolved_brands(self):
         # Basic test against Q4746

--- a/wikibrain/wikimedia_link_issue_reporter.py
+++ b/wikibrain/wikimedia_link_issue_reporter.py
@@ -682,7 +682,7 @@ class WikimediaLinkIssueDetector:
         dissolved_brands = self.get_dissolved_brands(present_wikidata_id.split(';'))
         if dissolved_brands:
             error_general_intructions = "Wikidata claims that assigned brand object no longer exists. That means that either this shop is gone or it is rebranded. Or brand:wikidata tag is wrong. Historical, no longer existing object should not be mapped in OSM." + " " + self.wikidata_data_quality_warning()
-            message = "wikidata ID " + ", ".join(dissolved_brands) + "is marked as dissolved"
+            message = "wikidata " + ", ".join(dissolved_brands) + " is marked as dissolved"
             state = "no longer existing brand (according to Wikidata)"
             if marked_as_defunct and not marked_as_active:
                 # see say https://www.openstreetmap.org/way/80889053

